pom.xml

```diff
@@ -10,16 +10,29 @@
   <!-- comment -->
   <!-- comment -->
 
+  <properties>
+    <jackson-version>2.9.9.3</jackson-version>
```

Comment:
### :thinking: Nexus IQ found policy violations introduced by:

<details open>
  <summary title="Threat Level: 10 of 10"><img alt="T10" src="https://cdn.sonatype.com/iq-for-scm/1.0/red-bar.png">
    <b>10&nbsp;&nbsp;&nbsp; com.fasterxml.jackson.core : jackson-databind : 2.9.9.3</b></summary>
<p></p>

:shield: **Bumping to version 2.10.0** will resolve these violations (as of _Nov 21, 2020_)

Threat (of 10) | Policy | Violation Details
--- | --- | --- |
10 | Security-Critical | <b>Critical risk CVSS score:</b><ul><li>Found security vulnerabilities: [CVE-2019-14540](http://localhost:8070/ui/links/vln/CVE-2019-14540), [CVE-2019-14892](http://localhost:8070/ui/links/vln/CVE-2019-14892), [CVE-2019-14893](http://localhost:8070/ui/links/vln/CVE-2019-14893), [CVE-2019-16335](http://localhost:8070/ui/links/vln/CVE-2019-16335), [CVE-2019-17267](http://localhost:8070/ui/links/vln/CVE-2019-17267)</li></ul>
9 | Security-High | <b>High risk CVSS score:</b><ul><li>Found security vulnerability: [sonatype-2019-0371](http://localhost:8070/ui/links/vln/sonatype-2019-0371)</li></ul>

</details>